cgi-bin/notify-email

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
```

Comment:
* I have quite strong feelings against bash scripts as cgi scripts.
* You are missing quotes around variables everywhere throughout the file.
* How is it supposed to be called via web?